Source/EasyNetQ/MessagePropertiesExtensions.cs

```diff
@@ -0,0 +1,80 @@
+using RabbitMQ.Client;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+
+namespace EasyNetQ;
+
+/// <summary>
+///     Represents various properties of a message
+/// </summary>
+internal static class MessagePropertiesExtensions
```

Comment:
Internal ? Did you do it intentionally? You removed 1 ctor and 2 methods from public API surface.